tests/integration/ipdevpoll/plugins/snmpcheck_test.py

```diff
@@ -21,37 +21,38 @@ def test_short_outage(localhost, db):
     job._create_agentproxy = Mock()
     job._destroy_agentproxy = Mock()
     agent.walk.return_value = defer.succeed(False)
+    start_events = EventQueue.objects.filter(
+        source_id='ipdevpoll',
+        target_id='eventEngine',
+        event_type='snmpAgentState',
+        netbox_id=localhost.pk,
+        state=EventQueue.STATE_START,
+    )
+    start_events_first_count = start_events.count()
+    end_events = EventQueue.objects.filter(
+        source_id='ipdevpoll',
+        target_id='eventEngine',
+        event_type='snmpAgentState',
+        netbox_id=localhost.pk,
+        state=EventQueue.STATE_END,
+    )
+    end_events_first_count = end_events.count()
```

Comment:
Ah, but it could be even nicer! :-)

```suggestion
    events = EventQueue.objects.filter(
        source_id='ipdevpoll',
        target_id='eventEngine',
        event_type='snmpAgentState',
        netbox_id=localhost.pk,
    )
    start_events = events.filter(state=EventQueue.STATE_START)
    end_events = events.filter(state=EventQueue.STATE_END)

    start_events_first_count = start_events.count()
    end_events_first_count = end_events.count()
```